app/models/manageiq/providers/base_manager.rb

```diff
@@ -29,6 +29,10 @@ def http_proxy_uri
       self.class.http_proxy_uri
     end
 
+    def console_url
+      raise NotImplementedError,, _("console_url must be implemented in a subclass")
```

Comment:
```suggestion
      raise NotImplementedError, _("console_url must be implemented in a subclass")
```

Reply:
Sorry just saw this was in my suggestion above, copy&paste error wanted to make sure I didn't get the exception name wrong and ended up doubling up on commas instead :facepalm: